src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -235,6 +236,13 @@ public function getRootNodeName()
         return false;
     }
 
+    private function appendComment(\DOMNode $node, string $data): bool
```

Comment:
Final protected like the others?

Reply:
Done ! (Could you explain to me the goal of a final protected method please ?)

Reply:
> goal of a final protected method

allowing child classes to call the method but not override it.